src/main/java/io/quarkus/registry/app/model/ExtensionRelease.java

```diff
@@ -45,7 +45,7 @@ SELECT DISTINCT ON(extension_release.extension_id)
                 extension_release
                 JOIN extension ON extension.id = extension_release.extension_id
                 LEFT JOIN platform_extension ON extension_release.id = platform_extension.extension_release_id
-                LEFT JOIN extension_release_compatibility ON extension_release_compatibility.extension_release_id = extension_release.id AND extension_release_compatibility.quarkus_core_version = :quarkusCore
+                LEFT JOIN extension_release_compatibility ON extension_release_compatibility.extension_release_id = extension_release.id AND :quarkusCore like extension_release_compatibility.quarkus_core_version
```

Comment:
I would recommend to test the queries with the production database as there's a good chance it was using an index before and won't be able to after this change.
What's the format of `extension_release_compatibility.quarkus_core_version` and what do you expect it to be after this change?

Reply:
I might miss something but I thought you were a lot more permissive for compatibility versions and that it was the point of the thing below with upper bounds and lower bounds?